test/membrane/rtp/stream_receive_bin_test.exs

```diff
@@ -102,4 +105,84 @@ defmodule Membrane.RTP.StreamReceiveBinTest do
     assert_end_of_stream(pipeline, :sink)
     Testing.Pipeline.terminate(pipeline, blocking?: true)
   end
+
+  defmodule NoopSource do
+    use Membrane.Source
+
+    def_output_pad :output, mode: :push, caps: :any
+
+    @impl true
+    def handle_event(:output, event, _ctx, state) do
+      {{:ok, notify: event}, state}
+    end
+  end
+
+  defmodule KeyframeRequester do
+    use Membrane.Sink
+
+    def_input_pad :input, demand_unit: :buffers, caps: :any
+
+    def_options delay: [spec: integer()]
+
+    @impl true
+    def handle_init(%{delay: delay}) do
+      {:ok, delay}
+    end
+
+    @impl true
+    def handle_prepared_to_playing(_ctx, delay) do
+      keyframe_request = {:event, {:input, %Membrane.KeyframeRequestEvent{}}}
+      Process.send_after(self(), keyframe_request, delay)
+      {{:ok, [keyframe_request, keyframe_request]}, delay}
+    end
+
+    @impl true
+    def handle_other(keyframe_request, _ctx, delay) do
+      {{:ok, [keyframe_request, keyframe_request]}, delay}
+    end
+  end
+
+  test "FIR sending with throttle" do
```

Comment:
in a lot of places we read tests as `test if x does something` but I am not sure if this the convention in rtp plugin